website/docs/src/content/cli-reference/deploy.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: gensx deploy
+description: Deploy a workflow to GenSX Cloud
+---
+
+# gensx deploy
+
+The `gensx deploy` command uploads and deploys a workflow to GenSX Cloud, making it available for remote execution.
+
+## Usage
+
+```bash
+gensx deploy <file> [options]
+```
+
+## Arguments
+
+| Argument | Description                                                                                |
+| -------- | ------------------------------------------------------------------------------------------ |
+| `<file>` | File to deploy. This should be a TypeScript/JavaScript file that exports a GenSX Workflow. |
+
+## Options
+
+| Option                  | Description                                                                  |
+| ----------------------- | ---------------------------------------------------------------------------- |
+| `-e, --env <KEY=value>` | Environment variable to include with deployment. Can be used multiple times. |
+| `-p, --project <name>`  | Project name to deploy to.                                                   |
+| `-h, --help`            | Display help for the command.                                                |
+
+## Description
+
+This command:
+
+1. Builds your workflow (if not already built)
+2. Uploads it to GenSX Cloud
+3. Creates or updates the deployment
+4. Sets up any environment variables specified
+
+After successful deployment, your workflow will be available for remote execution via the GenSX Cloud dashboard or through the `gensx run` command.
```

Comment:
Maybe add a link to the console? Also do we typically refer to it as a dashboard?